### PR TITLE
Passing intervals to s3 and localfile plugins

### PR DESCRIPTION
**Pull request by andreiko**
#### Summary
Fixed 2 bugs affecting S3 and localfile plugins:
- S3 plugin was using a hard-coded bucket name
- Both plugins didn't receive an interval value which was leading to division by zero and reporting all counter metrics as `+Inf`

#### Motivation
To fix the S3 plugin which is currently broken

#### Test plan
Modified a test to verify that a configured bucket name is used in S3Post()

### CODE REVIEW

**Comment by andreiko** [comment]
This is a mirror of the PR that I created in the upstream repo https://github.com/stripe/veneur/pull/379, but applied against v2.0.0 that we currently use.

File: plugins/localfile/localfile.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ type Plugin struct {
 	FilePath string
 	Logger   *logrus.Logger
 	hostname string
-	interval int
+	Interval float64
```
Comment (achille-roussel) [inline plugins/localfile/localfile.go]: It seems like this is a duration, why not use `time.Duration` here?

Comment (andreiko) [reply plugins/localfile/localfile.go]: It's only used for one thing - normalizing rates over the interval, expressed in seconds. I wanted these changes to be merged into upstream and tried to mirror how they do it in other sinks/plugins - https://github.com/segmentio/veneur/blob/master/sinks/datadog/datadog.go#L37

Comment (achille-roussel) [reply plugins/localfile/localfile.go]: Sounds good 👍

**Comment by andreiko** [comment]
Thanks @achille-roussel 👌

**Pull request merged**